README.md

```diff
@@ -56,14 +56,18 @@ These environment variables control the program's runtime:
 - `MINI_BATCH_NUMBER` - Number of mini-batches to process within each main batch. Used by `getTimeBatches` to divide the time between `prev_batch_end` and `cur_batch_end` into smaller intervals.
 - `UPTIME_DAYS_FOR_SCORE` - Number of days the system must be operational to calculate a score. Used by `updateScoreboard` to define the scoreboard update period.
 - `RETRY_COUNT` - Number of times a batch should be retried before giving up.
+- `SUBMISSION_STORAGE` - Storage where submissions are kept. Valid options: `POSTGRES` or `CASSANDRA`. Default: `POSTGRES`.
 
 ### Stateless Verification Tool Configuration
 
 The Coordinator program runs the `stateless-verification-tool` for validation against submissions. Set the following environment variables for this purpose:
 
-- `WORKER_IMAGE` - Docker image name for the stateless verifier (e.g., `mina-delegation-verify`).
+- `WORKER_IMAGE` - Docker image name for the stateless verifier (e.g., `delegation-verify`).
 - `WORKER_TAG` - Specific tag of the Docker image, indicating the version or build.
 - `NO_CHECKS` - if set to `1`, stateless verifier will run with `--no-checks` flag
+- `AWS_S3_BUCKET` - AWS S3 Bucket (needed for `stateless-verification-tool`)
+- `NETWORK_NAME` - Network name (needed for `stateless-verification-tool`, in case block does not exist in Cassandra 
```

Comment:
Now that you added postgres as alternativesyorage backend, perhaps text `in Cassandra` should change to storage backend or something more generic?